src/components/SearchBar/SearchBar.tsx

```diff
@@ -1,12 +1,48 @@
+import { useContext, useState } from "react";
 import SearchIcon from "../../assets/nav_bar_images/Group.svg";
-const SearchBar = (props: any) => {
+import { getQueriedData } from "../../api/api";
+// import { SearchContext } from "../context/SearchContext";
+
+
+const SearchBar = () => {
+  const [searchValue, setSearchValue] = useState("");
+  const [searchOptions, setSearchOptions] = useState([]);
+  // const setDataFromSearch = useContext(SearchContext);
+  // setDataFromSearch.setDataFromSearch([['ido'], [], []]);
+  const search = async (searchValue: string) => {
+    try {
+      const allData = await getQueriedData(searchValue);
+      console.log("Filtered data:", allData);
```

Comment:
remove unnecessary logs

Reply:
this is necessary as Dave asked me to do that,
when I search and press enter then I can see the search results